README.md

```diff
@@ -18,7 +18,7 @@ release of the geoarrow specification.
 ## Get started in Python
 
 ```python
-import geoarrow.pyarrow as ga
+import geoarrow.c.pyarrow as ga
```

Comment:
This made me wonder... can you do _nested_ namespace packages? Like I wonder if I could distribute

```
geoarrow.rs.core
geoarrow.rs.proj
geoarrow.rs.geos
geoarrow.rs.gdal
```
as four different pypi packages

Reply:
Just as a follow up, yes it works!

```
> tree
.
├── README.md
├── geoarrow
│   └── rs
│       └── core
│           ├── __init__.py
│           └── __pycache__
│               └── __init__.cpython-311.pyc
├── poetry.lock
└── pyproject.toml
```

```
> poetry run python
Python 3.11.4 (main, Aug 10 2023, 18:50:38) [Clang 14.0.3 (clang-1403.0.22.14.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import geoarrow
>>> import geoarrow.rs
>>> import geoarrow.rs.core
hello from geoarrow.rs.core
```